### PR TITLE
docs: dhis2 71 72

### PR DESCRIPTION
**Pull request by gnespolino**

### CODE REVIEW

File: src/commonmark/en/content/developer/web-api/new-tracker.md [hunk 1]
```diff
@@ -1,20 +1,346 @@
 # New Tracker
 
-  * Describe /tracker as a group of new tracker endpoints, where there are some new changes
-  * List changes we have made between 2.35->2.36
-  * Make a note that the old endpoints are marked as deprecated, but still work. Not all the functionality is ready in the new endpoint yet.
-
-## Tracker Import
+In version V2.36 DHIS2 introduced a set of brand new tracker endpoints dedicated to importing and querying tracker entities (tracked entity, enrollments, events, relationships, …).
```
Comment (enricocolasante) [inline src/commonmark/en/content/developer/web-api/new-tracker.md]: Do not need to put the dots at the end, there are only 4 tracker entities

File: src/commonmark/en/content/developer/web-api/new-tracker.md [hunk 2]
```diff
@@ -1,20 +1,346 @@
...
+
+These tables highlight V2.36 differences in input parameters for `GET` endpoints compared to V2.35
+
+#### Input parameters changes for `GET /api/tracker/enrollments`
+|V2.35|v2.36|
+|---|---|
+|`ou`|`orgUnit`|
+|`lastUpdated`|`lastUpdateDuration`|
+|`updatedAfter`|`updatedWithin`|
+|`programStartDate`|`programEndDate`|
```
Comment (enricocolasante) [inline src/commonmark/en/content/developer/web-api/new-tracker.md]: This parameter changed just the name or it is a different parameter?

File: src/commonmark/en/content/developer/web-api/new-tracker.md [hunk 3]
```diff
@@ -1,20 +1,346 @@
...
+|`enrolledAfter`|`enrolledBefore`|
```
Comment (enricocolasante) [inline src/commonmark/en/content/developer/web-api/new-tracker.md]: Same here?

**Review by enricocolasante — APPROVED** [review]
Some small comments.
I am just confused in the part of the new parameters for the get endpoints

File: src/commonmark/en/content/developer/web-api/new-tracker.md (hunk 2 above)
Comment (gnespolino) [reply src/commonmark/en/content/developer/web-api/new-tracker.md]: good catch, it's just bad table formatting, thank you!

File: src/commonmark/en/content/developer/web-api/new-tracker.md (hunk 3 above)
Comment (gnespolino) [reply src/commonmark/en/content/developer/web-api/new-tracker.md]: good catch, it's just bad table formatting, thank you!

File: src/commonmark/en/content/developer/web-api/new-tracker.md (hunk 1 above)
Comment (gnespolino) [reply src/commonmark/en/content/developer/web-api/new-tracker.md]: fair enough :)

**Pull request closed without merging**